astropy/coordinates/representation/geodetic.py

```diff
@@ -10,6 +11,37 @@
 from .cartesian import CartesianRepresentation
 
 
+class WestLongitudeMixin:
+    def to_cartesian(self):
+        self.lon = self.lon.value
+        return super().to_cartesian()
+
+    @classmethod
+    def from_cartesian(cls, cart):
+        angular = super().from_cartesian(cart)
+        return cls(
+            Longitude(
+                -1 * angular.lon.value,
+                u.deg,
+                wrap_angle=angular.lon.wrap_angle,
+                copy=False,
+            ),
+            angular.lat.to(u.deg),
+            angular.height,
+            copy=False,
+        )
+
+    @property
+    def lon(self):
+        return self._lon
+
+    @lon.setter
```

Comment:
This seems very risky; people should put in West longitudes! With my suggestions above, this is not necessary.